pkg/networking/networking_test.go

```diff
@@ -231,3 +252,125 @@ func Test_AddHeaders_AddsDefaultAndAuthHeaders(t *testing.T) {
 
 	assert.Equal(t, expectedHeader, request.Header)
 }
+
+func Test_AddUserAgent_AddsUserAgentHeaderToSnykApiRequests(t *testing.T) {
+	app := "snyk-ls"
+	appVersion := "20230508.144458"
+	osName := "DARWIN"
+	arch := "ARM64"
+	integrationName := "VS_CODE"
+	integrationVersion := "1.20.1"
+	integrationEnvironment := "language-server"
+	integrationEnvironmentVersion := "1.2.3.4"
+	processName := "snyk-ls"
+	expectedHeader := fmt.Sprint(
+		app, "/", appVersion,
+		" (", osName, ";", arch, ";", processName, ") ",
```

Comment:
Is `processName` an addition to what we discussed when discussing the ticket?

Reply:
No, it's part of the pattern (binary name)